routeros/resource_interface_ethernet.go

```diff
@@ -94,7 +90,7 @@ func ResourceInterfaceEthernet() *schema.Resource {
 				RX limit is supported only on Atheros8327/QCA8337 switch-chip ports.`,
 			DiffSuppressFunc: AlwaysPresentNotUserProvided,
 		},
-		"cable_settings": {
```

Comment:
What version of ROS are you testing the provider?
In v7.10 this field looks exactly like "cable-settings":
> Warning: Field 'cable_settings' not found in the schema

Reply:
I have to put aside some part of things and start versioning schemas. In fact, I don't update the docker container either, so as not to lose any changes in schemas....

Reply:
I don't have this particular one in my fleet. While adding the PoE fields, I noticed the difference in their docs. After double-checking and a quick search, I thought that was a typo on the provider's side. I reverted the rename. Sorry about that 😞 

> I have to put aside some part of things and start versioning schemas.

It shouldn't be necessary at this point. We can cover almost all of the cases with `AlwaysPresentNotUserProvided` rather than introducing an option (#271) requiring users to update that on every upgrade.

Reply:
@dokmic 
Please join the discussion #305